### PR TITLE
Bump to v5.6.0

### PR DESCRIPTION
**Pull request by ppeble**
Fixes https://github.com/holidays/holidays/issues/277

### CODE REVIEW

**Comment by ppeble** [comment]
Thanks! I'm moving fast because I've made @slucaskim wait. 😬

**Comment by ppeble** [comment]
Baaaaahh. Downloading jruby-9.0.5.0 to see what is going on. 😭

**Comment by ppeble** [comment]
Figured it out. Seems that jruby has different behavior for what comes back in terms of sorting than other rubies. I've just manually sorted to get this test to pass.

File: test/integration/test_holidays.rb [hunk 1]
```diff
@@ -229,21 +229,24 @@ def test_sub_regions
...
 
     # Should return Victoria Day and National Patriotes Day.
     holidays = Holidays.next_holidays(2, [:ca_], Date.civil(2008,5,1))
     assert_equal 2, holidays.length
-    assert_equal ['2008-05-19','Victoria Day'] , [holidays.first[:date].to_s, holidays.first[:name].to_s]
-    assert_equal ['2008-05-19','National Patriotes Day'] , [holidays.last[:date].to_s, holidays.last[:name].to_s]
+
+    # Aparently something in jruby doesn't sort the same way as other rubies so....we'll just do it ourselves so
+    # we don't flap.
+    sorted_holidays = holidays.sort_by { |h| h[:name] }
```
Comment (ttwo32) [inline test/integration/test_holidays.rb]: Good!! 👍

**Pull request merged**